hack/update-generated-files

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail -x
+
+: ${CONTINUOUS_INTEGRATION=}
+
+progressFlag=""
+if [ "$CONTINUOUS_INTEGRATION" == "true" ]; then progressFlag="--progress=plain"; fi
```

Comment:
This is not needed as there is no tty allocated on GHA. I think you can just remove this script